bears/xml2/XCopBear.py

```diff
@@ -0,0 +1,57 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GemRequirement import GemRequirement
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
+
+import re
+import itertools
+
+
+@linter(executable='xcop',
+        use_stdout=True)
+class XCopBear:
+    """
+    This bear can fix formatting errors in your XML file.
+    Check https://github.com/yegor256/xcop for more details.
```

Comment:
Please add a `SEE_MORE` attribute with this link. (Then you can actually omit the sentence here)

Reply:
After adding a `SEE_MORE` attribute do need to add in tests as well IMO.

Reply:
> After adding a SEE_MORE attribute do need to add in tests as well IMO.

I don't understand ;)

Reply:
What my question is as 'SEE_MORE' is doc testes are the test cases for SEE_MORE to be added in the XCOPtest.py?

Reply:
no ;)